src/components/switchTheme/useThemeContext.tsx

```diff
@@ -0,0 +1,46 @@
+import { createContext, ReactNode, useEffect, useState } from 'react';
+
+export interface ThemeContextType {
+  theme: 'light' | 'dark';
+  toggleTheme: () => void;
+}
+
+interface IProps {
```

Comment:
I know you know how to name it properly 🥇

Reply:
fixed